CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 See updating [Changelog example here](https://keepachangelog.com/en/1.0.0/)
 
 
+# 2.3.1 (21st August 2025)
+
+### Changed
+
+- Updating and fixing documentation
```

Comment:
I would like to see an entry noting that a refactor/movement of the Terraform code has taken place.  This is more to mark a point in time if there was a future issue.